app/views/evaluation/edit.html.haml

```diff
@@ -1,28 +1,71 @@
+%h1 Evaluation data:
+
 = form_tag evaluation_path(@evaluation), :method => :put, class: "form-horizontal" do
```

Comment:
Since this form shares a lot of the same design as the `new` form, it might be worth just pulling out that `new` form in to a partial and then using the partial in `new` and here in `edit`.